src/main/java/CommandDelete.java

```diff
@@ -0,0 +1,20 @@
+public class CommandDelete extends Command {
+    private final int index;
+
+    public CommandDelete (int index) {
+        this.index = index;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        Task task = tasks.deleteTask(index);
+        storage.save(tasks);
+        ui.printCommand(this);
+        ui.printTask(task);
+    }
+
+    @Override
+    public String toDukeOutput() {
+        return "Alrighty bossman. I shall wipe this task off the face of the earth: ";
```

Comment:
I also noticed that many one-line public methods from this point on also lack Javadoc comments. I believe that there should still be Javadoc for one-line methods to explain what the method intends to achieve.